scripts/esmDependencyCheck.js

```diff
@@ -0,0 +1,226 @@
+// eslint-disable no-console
+const { exec } = require('child_process');
+const fs = require('fs');
+const fetch = require('node-fetch');
+const dotenv = require('dotenv').config();
+const { dependencies, devDependencies } = require('../package.json');
+
+const allDependencies = { ...dependencies, ...devDependencies };
+const dependencyTable = [];
+
+const dateNow = new Date().getTime();
+const datediff = (first, second) =>
+  Math.round((second - first) / (1000 * 60 * 60 * 24));
+const gitToken = dotenv.parsed.GIT_TOKEN || false;
```

Comment:
As mentioned above, I think we should use environment variables here.
If it is not populated, the value will be undefined, therefore no need for `|| false`
```suggestion
const gitToken = process.env.GITHUB_TOKEN
```

Reply:
Changed to this